src/platform/user/profile/utilities/index.js

```diff
@@ -42,9 +41,6 @@ export function mapRawUserDataToState(json) {
     gender,
     isVeteran,
     loa,
-    mhv: {
-      terms: { accepted: healthTermsCurrent }
-    },
```

Comment:
Removed because everything will be in the `mhvAccount` object, which gets populated by the `fetchMHVAccount` action wherever that data is needed.